tests/test_z_defaults.py

```diff
@@ -50,7 +50,7 @@ def collect_snippets(path):
     return Hedy_snippets
 
 
-Hedy_snippets = [(s.name, s) for s in collect_snippets(path='../coursedata/level-defaults')]
+Hedy_snippets = [(s.name, s) for s in collect_snippets(path=utils.construct_content_path('level-defaults'))]
```

Comment:
Ah this is the offender "destroying" the Arabic snippets, we rely on the path name to find the lang. Will fix!